common/models/ClientCard.ts.template

```diff
@@ -44,7 +44,12 @@ export namespace CoachClientActions {
     export const Helper = new EnumStringHelper<CoachClientActions>(CoachClientActions);
 }
 
-export const ClientInviteMinPeriod = 1000 * 3600 * 24; // 1 day
+const clientInvitePeriods = {
+    DEFAULT: 1000 * 3600 * 24,
+    TEST: 1000,
+};
+
+export const ClientInviteMinPeriod = clientInvitePeriods.${CLIENT_INVITE_PERIOD};
```

Comment:
The clientInviteMinPeriod flag works great. When testing it I noticed the below message when deploying. I was able to just choose to continue deployment, but have you seen this? 

<img width="567" alt="Screen Shot 2021-02-09 at 6 26 02 PM" src="https://user-images.githubusercontent.com/47067364/107457444-ec197800-6b06-11eb-8a42-fd1dad45776a.png">

Reply:
@cara-wong yeah that's a function I deployed for testing my export functions on a different branch! I think it's just a warning that your local source code is not in sync with the deployed project.